[2023-04-09T00:45:20.726+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.process_user manual__2023-04-09T00:40:11.931406+00:00 [queued]>
[2023-04-09T00:45:20.734+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: api_dag.process_user manual__2023-04-09T00:40:11.931406+00:00 [queued]>
[2023-04-09T00:45:20.735+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:45:20.739+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 2
[2023-04-09T00:45:20.739+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-09T00:45:20.749+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): process_user> on 2023-04-09 00:40:11.931406+00:00
[2023-04-09T00:45:20.755+0000] {standard_task_runner.py:55} INFO - Started process 10381 to run task
[2023-04-09T00:45:20.760+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'api_dag', 'process_user', 'manual__2023-04-09T00:40:11.931406+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/api.py', '--cfg-path', '/tmp/tmpgqu17yt0']
[2023-04-09T00:45:20.764+0000] {standard_task_runner.py:83} INFO - Job 93: Subtask process_user
[2023-04-09T00:45:20.827+0000] {task_command.py:376} INFO - Running <TaskInstance: api_dag.process_user manual__2023-04-09T00:40:11.931406+00:00 [running]> on host c12a518d534c
[2023-04-09T00:45:20.880+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=api_dag
AIRFLOW_CTX_TASK_ID=process_user
AIRFLOW_CTX_EXECUTION_DATE=2023-04-09T00:40:11.931406+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-09T00:40:11.931406+00:00
[2023-04-09T00:45:20.898+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-04-09T00:45:20.908+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=api_dag, task_id=process_user, execution_date=20230409T004011, start_date=20230409T004520, end_date=20230409T004520
[2023-04-09T00:45:20.931+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-09T00:45:20.955+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
